www/in-the-wild.html

```diff
@@ -408,7 +408,6 @@ <h4><a name="tools"></a>Tools</h4>
     <li><a href="https://micr.io">Micrio</a> (<a href="https://micr.io/iiif/image-api">Demo</a>)</li>
     <li><a href="http://projectmirador.org">Mirador</a> (<a href="http://projectmirador.org/demo/">demo</a> - <a href="https://github.com/IIIF/mirador">GitHub</a>)</li>
     <li><a href="http://mosaically.com/">Mosaically</a></li>
-    <li><a href="https://github.com/whalehub/noita-map-viewer">Noita Map Viewer</a> (<a href="https://noita.datahoarder.dev">Demo</a>)</li>
```

Comment:
I'm not sure why this is getting deleted as part of this patch, but it shouldn't be. It's a newly-merged item (from yesterday), so it probably just needs you to merge the latest into your fork.

Reply:
Note that this project is about games as well but I think it still belongs in the Tools section.